Project/QtCreator/qctools-gui/qctools-gui.pro

```diff
@@ -208,20 +205,16 @@ INCLUDEPATH += $$SOURCES_PATH
 INCLUDEPATH += $$SOURCES_PATH/ThirdParty/cqmarkdown
 include(../ffmpeg.pri)
 
-win32-msvc* {
-    LIBS += $${THIRD_PARTY_PATH}/zlib/lib/zlibstatic.lib
-}
-
 win32-g++* {
-    LIBS += -lz -lbcrypt -lwsock32 -lws2_32
+    LIBS += -lbcrypt -lwsock32 -lws2_32
 }
 
 !win32 {
     LIBS      += -lbz2
```

Comment:
It's not related to your changes, but is it really necessary? I haven't removed that because I thought it could be useful to someone else.